tests/service/PatternMatchCompilationTests.fs

```diff
@@ -1260,4 +1260,20 @@ let f : obj -> _ =
     assertHasSymbolUsages ["i"] checkResults
     dumpDiagnostics checkResults |> shouldEqual [
         "(5,6--5,18): Feature 'non-variable patterns to the right of 'as' patterns' is not available in F# 5.0. Please use language version 6.0 or greater."
-    ]
\ No newline at end of file
+    ]
+
+[<Test>]
+#if !NETCOREAPP
+[<Ignore("These tests weren't running on desktop and this test fails")>]
```

Comment:
I see this thing is here for other tests in this file, but do you happen to know what's this about? (why would they fail on desktop?)

Reply:
Something was wrong with the checking, I can't remember what exactly, unfortunately.

Reply:
Hah, same question at the same time :eyes: 
Well anyway, if this new test works for desktop, feel free to enable it.

Reply:
Okay looks like it works everywhere, I tried and pushed, resolving, merging :)